app/controllers/admin/bookings_controller.rb

```diff
@@ -28,7 +28,26 @@ def load_booking
     redirect_to admin_bookings_path
   end
 
+  def search_bookings
+    start_at = format_date params[:start_at]
+    created_at = format_date params[:created_at]
+    booking_status = params[:booking_status]
+
+    @bookings = if start_at.blank? && created_at.blank? && booking_status.blank?
+                  Booking.newest
+                else
+                  Booking.search_by_booking_status(booking_status)
+                         .search_by_start_at(start_at)
+                         .search_by_created_at(created_at)
+                         .newest
+                end
```

Comment:
```suggestion
    @bookings = Booking.search_by_booking_status(booking_status)
                         .search_by_start_at(start_at)
                         .search_by_created_at(created_at)
                         .newest
```
chị thấy em đã check param present trong scope rồi nên ko cần if else đonạ ni nha

Reply:
ok c ơi. task sau e refactor luôn